packages/validator/src/services/indices.ts

```diff
@@ -14,6 +14,26 @@ const PUBKEYS_PER_REQUEST = 10;
 // To assist with readability
 type PubkeyHex = string;
 
+// To assist with logging statuses, we only log the statuses that are not active_exiting or withdrawal_possible
+type SimpleValidatorStatus = "pending" | "active" | "exited" | "withdrawn" | "unknown";
+
+const statusToSimpleStatusMapping = (status: routes.beacon.ValidatorStatus): SimpleValidatorStatus => {
+  if (status.includes("active") || status === "withdrawal_possible") {
+    return "active";
+  }
+  if (status.includes("exiting")) {
```

Comment:
There is no "exiting_*" ValidatorStatus... It may be better to just use a switch case here